onnx/test/optimizer_test.py

```diff
@@ -194,12 +199,17 @@ def check_pad(node):  # type: (NodeProto) -> None
         assert len(optimized_model.graph.node) == 0
 
     def test_nop_pad_default(self):  # type: () -> None
-        trans = helper.make_node("Pad", ["X"], ["Y"], pads=[0, 1])
+        trans = helper.make_node("Pad", ["X", "Pads"], ["Y"])
         graph = helper.make_graph(
             [trans],
             "test",
-            [helper.make_tensor_value_info("X", TensorProto.FLOAT, (2, 3))],
-            [helper.make_tensor_value_info("Y", TensorProto.FLOAT, (2, 4))])
+            [helper.make_tensor_value_info("X", TensorProto.FLOAT, (2, 3)),
+             helper.make_tensor_value_info("Pads", TensorProto.INT64, (4,))],
+            [helper.make_tensor_value_info("Y", TensorProto.FLOAT, (2, 4))],
+            [helper.make_tensor("Pads", TensorProto.INT64,
+                dims=(4,),
+                vals=np.array([0, 1, 0, 0]).astype(np.int64).tobytes(),
+                raw=True)])
         optimized_model = self._optimized(graph, ["eliminate_nop_pad"])
```

Comment:
I guess this `Pad` won't be removed. If yes, could you rename this test to `not_eliminate_pad`?

Reply:
Good point. will do.

Reply:
Done. thanks for catching this.

Reply:
by the way - the line your comment is pointing to is the optimizer pass name - that cannot be modified. but the test name can be modified and I modified that.